### ISSUE TITLE
Your test performance

### ISSUE THREAD

**Issue by GabrielLin** [Determined22/zh-NER-TF#4]
Could you please shard your test performance? I cannot load your pre-trained model and want to compare of the model I train myself. Thanks.

**Comment by GabrielLin** [comment]
=========== testing ===========
INFO:tensorflow:Restoring parameters from ./data_path_save/1512814965/checkpoints/model-47520
Restoring parameters from ./data_path_save/1512814965/checkpoints/model-47520
2017-12-10 01:16:04.026702: I tensorflow/core/common_runtime/gpu/pool_allocator.cc:247] PoolAllocator: After 25445 get requests, put_count=25405 evicted_count=1000 eviction_rate=0.0393623 and unsatisfied allocation rate=0.0448025
2017-12-10 01:16:04.026781: I tensorflow/core/common_runtime/gpu/pool_allocator.cc:259] Raising pool_size_limit_ from 100 to 110
processed 177232 tokens with 6192 phrases; found: 6029 phrases; correct: 5428.
accuracy:  98.58%; precision:  90.03%; recall:  87.66%; FB1:  88.83
LOC: precision:  92.83%; recall:  90.48%; FB1:  91.64  2804
ORG: precision:  85.60%; recall:  85.73%; FB1:  85.66  1333
PER: precision:  89.01%; recall:  84.88%; FB1:  86.89  1892
